.bazelrc.common

```diff
@@ -58,8 +60,6 @@ build:ci-common --repository_cache=~/repo-cache/
 build:ci-common --experimental_repository_cache_hardlinks
 
 # Use a remote cache during CI
-build:ci-common --bes_results_url=https://app.buildbuddy.io/invocation/
-build:ci-common --bes_backend=grpcs://remote.buildbuddy.io
```

Comment:
Maybe it would make sense to still leverage the cache in general, but disable remote uploads with `--noremote_upload_local_results`, and re-enable remote uploads for the "ci" config with `--remote_upload_local_results`. Not sure this is possible.